config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatatransferconfigs.bigquerydatatransfer.cnrm.cloud.google.com.yaml

```diff
@@ -123,128 +146,87 @@ spec:
+                      to a recurrence pattern defined in the schedule string. The
+                      start time can be changed at any moment. The time when a data
+                      transfer can be triggered manually is not limited by this option.
                     type: string
                 type: object
-              sensitiveParams:
-                description: |-
-                  Different parameters are configured primarily using the the 'params' field on this
-                  resource. This block contains the parameters which contain secrets or passwords so that they can be marked
-                  sensitive and hidden from plan output. The name of the field, eg: secret_access_key, will be the key
-                  in the 'params' map in the api request.
-
-                  Credentials may not be specified in both locations and will cause an error. Changing from one location
-                  to a different credential configuration in the config will require an apply to update state.
-                properties:
-                  secretAccessKey:
-                    description: The Secret Access Key of the AWS account transferring
-                      data from.
-                    oneOf:
-                    - not:
-                        required:
-                        - valueFrom
-                      required:
-                      - value
-                    - not:
-                        required:
-                        - value
-                      required:
-                      - valueFrom
-                    properties:
-                      value:
-                        description: Value of the field. Cannot be used if 'valueFrom'
-                          is specified.
-                        type: string
-                      valueFrom:
-                        description: Source for the field's value. Cannot be used
-                          if 'value' is specified.
-                        properties:
-                          secretKeyRef:
-                            description: Reference to a value with the given key in
-                              the given Secret in the resource's namespace.
-                            properties:
-                              key:
-                                description: Key that identifies the value to be extracted.
-                                type: string
-                              name:
-                                description: Name of the Secret to extract a value
-                                  from.
-                                type: string
-                            required:
-                            - name
-                            - key
-                            type: object
-                        type: object
-                    type: object
-                required:
-                - secretAccessKey
-                type: object
-              serviceAccountName:
```

Comment:
This is also a terraform only field.
https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/bigquery_data_transfer_config#service_account_name